### DISCUSSION TITLE
Need suggestion for static landing page

### DISCUSSION THREAD

**Discussion by SuryodayYadav** [egovernments/Digit-Core#91] [Announcements]
How to build a static landing page on UPYOG platform ? Is it making a new module itself or do i need to make a folder outside the module? if it is yes then how do i configure the page inside the UPYOG platform? 

Note: That page will have 7 different module, that i can take care of by making new modules, but where to insert the landing page that is confusing.

**Comment by jagankumar-egov** [comment]
For **Employee**, a landing screen card can be created by referring to this 
https://core.digit.org/guides/developer-guide/ui-developer-guide/employee-module-setup/write-employee-module-code#employee-module-card

For **citizen** , a landing screen card can be created by adding mdms change referring to this 
https://core.digit.org/guides/developer-guide/ui-developer-guide/citizen-module-setup/citizen-landing-screen#how-to-add-a-new-card
